piquasso/_math/torontonian.py

```diff
@@ -0,0 +1,15 @@
+# Placeholder for torontonian and loop_torontonian functions required by piquasso._simulators.gaussian.probabilities
```

Comment:
What is this file, why is it required?

Reply:
OK, so these are not required in the production code. In the package, you already have these implementations compiled, you don't need any placeholder, it will break the code. You can use these stubs for development, but don't commit them here. I also encourage you to compile Piquasso as described in the `README.md`.